tools/registry.c

```diff
@@ -157,7 +159,23 @@ main( int argc, char *argv[], char *env[] )
       sprintf( fname_wrk,"%s/Registry_irr_diag",dir ) ;
     }
 //  fprintf(stderr,"Registry tmp file = %s\n",fname_wrk);
-    sprintf(command,"/bin/cp %s %s\n",fname_in,fname_wrk);
+    /* we should be able to implement this using posix_spawn */
```

Comment:
Remove unless this is for future use/notes

Reply:
It's notes for a potential future application. My usual development system doesn't have a working `posix_spawn`.  I could try a regular `spawn`, but then I'd have to test for whether the system has that function and I'm not sure how to do that.

I'd forgotten about this until you reminded me.

Reply:
Removed to a different branch for consideration later.